src/randomenv.cpp

```diff
@@ -57,8 +57,12 @@
 #include <sys/auxv.h>
 #endif
 
-#ifndef WIN32
-extern char** environ; // NOLINT(readability-redundant-declaration): Necessary on some platforms
```

Comment:
nit: I think the comment can be kept as-is, or just refer to "Necessary on the above platforms". Otherwise, it is just one more line to maintain.

Reply:
Re-modified the comment. Not sure there is too much value in moving this to configure.